maat-court-data-api/src/main/java/gov/uk/courtdata/dces/controller/DebtCollectionController.java

```diff
@@ -0,0 +1,41 @@
+package gov.uk.courtdata.dces.controller;
+
+import com.amazonaws.xray.spring.aop.XRayEnabled;
+import gov.uk.courtdata.dces.service.DebtCollectionService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@RestController
+@RequestMapping("${api-endpoints.debt-collection-enforcement}")
+@Slf4j
+@RequiredArgsConstructor
+@XRayEnabled
+@Tag(name = "Debt Collection Enforcement", description = "Rest API for Debt Collection Enforcement Service")
+public class DebtCollectionController {
+
+    private final DebtCollectionService dceService;
+
+    @GetMapping(value = "/contributions", produces = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(description = "Get a list of XML files for contributions ")
+    public List<String> getContribution(@RequestParam(name = "fromDate") final LocalDate fromDate,
+                                        @RequestParam(name = "toDate") final LocalDate toDate) {
+        log.info("Get correspondence status request for getContribution with date range from {} to {}", fromDate, toDate);
+        return dceService.getContributionFiles(fromDate, toDate);
+    }
+
+    @GetMapping(value = "/final-defence-cost", produces = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(description = "Get a list of XML files for FDC (final-defence-cost).")
+    public List<String> getFDC(@RequestParam(name = "fromDate") final LocalDate fromDate,
```

Comment:
Is there a specific format that the date parameter needs to be specified in?

Reply:
In the database, we have this format 25-FEB-21 

This is something that needs to be looked into so that the API should accept it in a specific format. Currently, it's accepted as an ISO type.